Vungle/VungleRouter.m

```diff
@@ -683,25 +757,26 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
                 }
 
                 if (needToClearDelegate) {
-                    [self clearDelegateWithState:BannerRouterDelegateStateClosed
-                                     placementID:nil];
+                    [self clearBannerDelegateWithState:BannerRouterDelegateStateClosed];
                 }
             }
         }
     }
 }
 
 - (void)vungleWillShowAdForPlacementID:(nullable NSString *)placementID
+                               eventId:(NSString*)eventID
 {
     if (!placementID.length) {
         return;
     }
-
-    id<VungleRouterDelegate> targetDelegate = [self.delegatesDict objectForKey:placementID];
+    
```

Comment:
Might there be an extra indent? Thanks a lot :)

Reply:
Updated.